### PR TITLE
Typos: "re-use", "a same".

### PR DESCRIPTION
**Pull request by ozabluda**

### CODE REVIEW

File: docs/templates/getting-started/functional-api-guide.md [hunk 1]
```diff
@@ -256,7 +256,7 @@ assert lstm.get_output_at(1) == encoded_b
 
 Simple enough, right?
 
-The same is true for the properties `input_shape` and `output_shape`: as long as the layer has only one node, or as long as all nodes have the same input/output shape, then the notion of "layer output/input shape" is well defined, and that one shape will be returned by `layer.output_shape`/`layer.input_shape`. But if, for instance, you apply a same `Conv2D` layer to an input of shape `(32, 32, 3)`, and then to an input of shape `(64, 64, 3)`, the layer will have multiple input/output shapes, and you will have to fetch them by specifying the index of the node they belong to:
+The same is true for the properties `input_shape` and `output_shape`: as long as the layer has only one node, or as long as all nodes have the same input/output shape, then the notion of "layer output/input shape" is well defined, and that one shape will be returned by `layer.output_shape`/`layer.input_shape`. But if, for instance, you apply the same `Conv2D` layer to an input of shape `(32, 32, 3)`, and then to an input of shape `(64, 64, 3)`, the layer will have multiple input/output shapes, and you will have to fetch them by specifying the index of the node they belong to:
```
Comment (fchollet) [inline docs/templates/getting-started/functional-api-guide.md]: What's the different between these lines?

Comment (bdwyer2) [reply docs/templates/getting-started/functional-api-guide.md]: you apply **a** same `Conv2D` --> you apply **the** same `Conv2D`

**Comment by fchollet** [comment]
Neither fixes are actual typos, "re-use" is a valid spelling and "a same layer" is the intended form.

**Comment by ozabluda** [comment]
@fchollet, Oops, sorry about "re-use", it's a correct spelling, but very rare, not even every dictionary lists it as correct. For example OED doesn't. 
https://www.google.com/search?q=re-use+or+reuse

But "a same" seems completely wrong (or at best extremely exotic):
https://www.google.com/search?q=a+same+vs+the+same

**Comment by fchollet** [comment]
Ok, that's reasonable

**Pull request merged**